doc/modules/model_evaluation.rst

```diff
@@ -1112,6 +1113,103 @@ function:
     >>> matthews_corrcoef(y_true, y_pred)  # doctest: +ELLIPSIS
     -0.33...
 
+.. _multilabel_confusion_matrix:
+
+Multi-label confusion matrix
+----------------------------
+
+The :func:`multilabel_confusion_matrix` function computes class-wise (default)
+or sample-wise (samplewise=True) multilabel confusion matrix to evaluate
+the accuracy of a classification. multilabel_confusion_matrix also treats
+multiclass data as if it were multilabel, as this is a transformation commonly
+applied to evaluate multiclass problems with binary classification metrics
+(such as precision, recall, etc.).
+
+When calculating class-wise multilabel confusion matrix :math:`MCM`, the
+count of true negatives for class :math:`i` is :math:`MCM_{i,0,0}`, false
+negatives is :math:`MCM_{i,1,0}`, true positives is :math:`MCM_{i,1,1}`
+and false positives is :math:`MCM_{i,0,1}`.
+
+Here is an example demonstrating the use of the
+:func:`multilabel_confusion_matrix` function with
+:term:`multilabel indicator matrix` input::
+
+    >>> import numpy as np
+    >>> from sklearn.metrics import multilabel_confusion_matrix
+    >>> y_true = np.array([[1, 0, 1],
+    ...                    [0, 1, 0]])
+    >>> y_pred = np.array([[1, 0, 0],
+    ...                    [0, 1, 1]])
+    >>> multilabel_confusion_matrix(y_true, y_pred)
+    array([[[1, 0],
+            [0, 1]],
+    <BLANKLINE>
+           [[1, 0],
+            [0, 1]],
+    <BLANKLINE>
+           [[0, 1],
+            [1, 0]]])
+
+Here is an example demonstrating the use of the
+:func:`multilabel_confusion_matrix` function with
+:term:`multiclass array` input::
```

Comment:
"multiclass array" isn't a term in the [Glossary](https://github.com/scikit-learn/scikit-learn/blob/master/doc/glossary.rst). "multiclass" is.